translations/stripes-erm-components/en.json

```diff
@@ -15,7 +15,11 @@
   "errorLogNo": "No errors to report.",
   "columns.errorLogMessage": "Error",
   "columns.infoLogMessage": "Message",
-  "columns.recordNumber": "Record number",
+  "columns.recordDescriptor": "Record descriptor",
```

Comment:
Its best to leave an empty line between lines 18 and 19 just to differentiate the related translations